action.yml

```diff
@@ -53,10 +53,10 @@ runs:
     - name: Mount devbox cli cache
       if: inputs.refresh-cli == 'false'
       id: cache-devbox-cli
-      uses: actions/cache@v3
+      uses: actions/cache/restore@v3
       with:
         path: /usr/local/bin/devbox
-        key: ${{ runner.os }}-devbox-${{ env.latest_version }}
+        key: ${{ runner.os }}-devbox-cli-${{ env.latest_version }}
```

Comment:
I vote for not caching the cli. Do we really save anything? Downloading from s3 is basically a cache already.

Removing it makes the action simpler.

Reply:
I think the devbox launcher is fetching the actual binary from Github. When actions are running in parallel, we would hit the github rate limit.